tests/manual/mention-custom-view.md

```diff
@@ -0,0 +1,39 @@
+## Mention
+
+**NOTE**: Playground for testing model implementation.
+
+Use the code of this manual sample to change bahavior of the feature.
+
+### Attribute (default)
+- works kinda OK - should use objects to store mention data
```

Comment:
There's too much casual language and too few testing here in this MD file. Let's make a proper test description out of it?